### ISSUE TITLE
Maximize maximizing beyond screen size.

### ISSUE THREAD

**Issue by xMesa-git** [zhiyiYo/PyQt-Frameless-Window#96]
**Describe the bug**
When i maximize my tool, the right and bottom limits on my window go outside of the limits of the screen.

**Environment**
 - OS: Windows10
 - DPI scaling: 100%
 - Python: 3.9.13 64-bit
 - PySide6: 6.5.0
 **- pywin32: 306**

**To Reproduce**
When i drag-and-drop on side-of-screen it works ok:
But when i maximize, the close button (red x) is half outside of the screen.
Same for the bottom part of the app.

**Expected behavior**
Apps maximize to available screen size.

**Screenshots**
![image](https://github.com/zhiyiYo/PyQt-Frameless-Window/assets/59180701/0555619b-a69b-47ba-b61c-c2a4c4b1cec7)

Unfortunately i cannot upload a gif, as the company policy is against this file type :-/

I tried playing with 

```python
            # adjust the size of client rect
            if isMax and not isFull:
                ty = win_utils.getResizeBorderThickness(msg.hWnd, False)
                rect.top += ty
                rect.bottom += ty

                tx = win_utils.getResizeBorderThickness(msg.hWnd, True)
                rect.left += tx
                rect.right += tx
```

but no success.

Any suggestion where i might check?

**Comment by zhiyiYo** [comment]
use 6.4.2

**Comment by xMesa-git** [comment]
There is an incompatibility between PySide6.4.2 and pywin32, the same one you described here:

https://stackoverflow.com/questions/58612306/how-to-fix-importerror-dll-load-failed-while-importing-win32api/72488468#72488468

But this is fixed in 6.4.3 & further. So any other tips on how to fix this?
